cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRMutationFieldSetMapper.java

```diff
@@ -43,24 +43,144 @@
  *
  * @author heinsz
  */
-public class CVRMutationFieldSetMapper implements  FieldSetMapper<MutationRecord> {
-    Logger log = Logger.getLogger(CVRMutationFieldSetMapper.class);
+public class CVRMutationFieldSetMapper implements FieldSetMapper<MutationRecord> {
+    private static final Logger log = Logger.getLogger(CVRMutationFieldSetMapper.class);
+
     @Override
     public MutationRecord mapFieldSet(FieldSet fs) throws BindException {
         MutationRecord record = new MutationRecord();
-        Set<String> names = new HashSet(Arrays.asList(fs.getNames()));
-        names.addAll(record.getHeader());
+        Set<String> names = new HashSet<>(Arrays.asList(fs.getNames()));
+
         for (String field : names) {
-            try {
```

Comment:
I got rid of additionalPropertyKeys.  It was being used to make sure all rows had their additionalproperties populated with the same set of fields.  This wasn’t needed because I think the writers just used getHeader to create a header.  Then when the getter is called on each field, if the property doesn’t exist in additional properties, an empty string is returned there.  TODO check this is correct.